tests/tests-c-compiler/check-src/check-126.-gen-PER.c

```diff
@@ -65,7 +66,12 @@ save_object_as(PDU_t *st, enum enctype how) {
 	 * Save object using specified method.
 	 */
 	switch(how) {
-	case AS_PER:
+	case AS_APER:
```

Comment:
I think that `AS_PER` should be the same as `AS_UPER`, **not** as `AS_APER`. This case selector should be moved down to before line 74, I think.